mod/phpmailer/start.php

```diff
@@ -250,11 +250,17 @@ function phpmailer_send($to, $to_name, $subject, $body, array $bcc = NULL, $html
 		$errType = "custom";
 		phpmailer_logging($errMess, $errStack, 'PHPMailer', $errType);
 	}
-
+	count_emails();
 	return $return;
 }
 
+function count_emails(){
```

Comment:
This is vendor'd in now, but is there a way to do this without modifying the 3rd party code through hooks instead?

Reply:
Pretty sure only the core elgg mail functionality has a hook but I don't think it would be all that useful with the whole notification / email system being handled by the cp_notifications mod (and missions mod does its own thing I think, will need to add counts there too).